src/Subscriber/FrontendSubscriber.php

```diff
@@ -46,7 +45,7 @@ public function onHeaderLoaded(HeaderPageletLoadedEvent $event): void
 
     public function onKernelResponse(ResponseEvent $event): void
     {
-        if (!($event->getResponse() instanceof StorefrontResponse)) {
+        if (!($event->getResponse())) {
```

Comment:
Why was the type check removed?

Reply:
The StorefrontResponse class does not exist in the core for SW6.6. I also made a comparison between the core for SW6.5 and SW6.6, and in places where such a check (instanceof StorefrontResponse) was performed, it no longer exists for SW6.6. Here’s an example:
![image](https://github.com/user-attachments/assets/f9f64b44-1ed8-4af1-b47f-d3c092bd1314)
![image](https://github.com/user-attachments/assets/59ac7557-c057-4c61-ba80-1ae9dfd3a54b)